qa/workunits/rbd/krbd_watch_errors_exclusive.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -ex
+set -o pipefail
+
+readonly IMAGE_NAME="watch-errors-exclusive-test"
+
+rbd create -s 1G --image-feature exclusive-lock,object-map "${IMAGE_NAME}"
+
+# induce a watch error every 30 seconds
+dev="$(sudo rbd device map -o exclusive,osdkeepalive=60 "${IMAGE_NAME}")"
```

Comment:
I see this on other rbd workunits as well. How does this induce watch error every 30 seconds?

Reply:
This is an implementation detail of the kernel client -- `osdkeepalive=60` makes the periodic work that, among other things, is responsible for pinging the watch to prove to the OSD that the client is still alive run once every 60 seconds instead of every 5 seconds.  The OSD then times out the watch.

Reply:
Correct.